modules/createHashHistory.js

```diff
@@ -327,6 +327,19 @@ const createHashHistory = (props = {}) => {
     };
   };
 
+  const link = (path, state) => {
+    const currentLocation = {
+      path: history.location.pathname + history.location.search + history.location.hash,
+      state: history.state || {}
+    };
+
+    if (shouldReplace(currentLocation, {path})) {
```

Comment:
Should `state` not also be passed to this method, as it's in the params?

Reply:
There is no state in hash history

Reply:
Then remove it from the method params & default data?